src/apollo/queries.js

```diff
@@ -1,5 +1,124 @@
 import gql from 'graphql-tag';
 
+export const UNISWAP_PAIRS_ID_QUERY = gql`
+  query pairs {
+    pairs(first: 200, orderBy: trackedReserveETH, orderDirection: desc) {
+      id
+    }
+  }
+`;
+
+export const UNISWAP_PAIR_DATA_QUERY = (pairAddress, block) => {
+  const queryString = `
```

Comment:
To help remove bulk in our responses, can remove unused params from this query:
- reserveETH
- txCount
- token0Price
- token1Price
- createdAtTimestamp

Reply:
Good call! Done